### PR TITLE
Add test API + 2 common utils

### PR DESCRIPTION
**Pull request by Mihullo**
Added:
1. Test suite of API at https://reqres.in/
2. Two common utilities: colored_messages, extract_from_json
3. Package name 'Fake' in requirements.txt

### CODE REVIEW

File: Utils/colored_messages.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""Yellow text on red background"""
+
+
+def yellow_on_red_back(text):
+    return f'\033[33m\033[41m\033[6m\033[1m >>> {text} <<< \033[0m'
```
Comment (BurhanH) [inline Utils/colored_messages.py:5]: Interesting!

File: requirements.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 requests==2.27.1
+Faker
```
Comment (BurhanH) [inline requirements.txt:2]: Do we need specify a version of faker library?

File: tests/test_full_reqres_api_MiK.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+HTTP_STATUS_OK = 200
+HTTP_STATUS_CREATE = 201
+HTTP_STATUS_DELETE = 204
+HTTP_STATUS_UNSUCCESSFUL = 400
+HTTP_STATUS_NOT_FOUND = 404
+
+TEST_DATA = {
+    'name': Fake().name(),
+    'job':  Fake().job(),
```
Comment (BurhanH) [inline tests/test_full_reqres_api_MiK.py:26]: Nice! Using faker to generate test data!

File: tests/test_full_reqres_api_MiK.py [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+}
+
+
+class TestReqresAPI(unittest.TestCase):
+
+    @pytest.mark.reqres
+    def test_1_check_list_users_pages(self):
+        """Getting the default list of users. Determining the number of pages and users."""
+        response = requests.get(f'{BASE_URL}{ALL_USERS}')
+        self.assertEqual(response.status_code, HTTP_STATUS_OK, bug_report('Bad request total user page.'))
```
Comment (BurhanH) [inline tests/test_full_reqres_api_MiK.py:47]: Now I got it! If test fail we will get a colored error message in terminal. Nice!

File: tests/test_full_reqres_api_MiK.py [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+        total_user_pages = (response.json())['total_pages']
+        self.assertGreater(total_user_pages, 0, bug_report('The user database is not filled.'))
+        total_users = (response.json())['total']
```
Comment (BurhanH) [inline tests/test_full_reqres_api_MiK.py:50]: Good!

**Review by BurhanH — COMMENTED** [review]
Please review my comments!

**Comment by BurhanH** [comment]
For some reason your PR has failures.
Please fix them!
Thanks @Mihullo

File: requirements.txt (hunk 2 above)
Comment (Mihullo) [reply requirements.txt:2]: I think it would be better to fix the version for the stability of the framework. I will mark the current version 13.13.0.

**Comment by Mihullo** [comment]
**Fixed!**
Added: conftest.py -- Without it, Pytest does not see the 'Utils' package;
Deleted: __init.py__ from tests\ -- With it, PyTest does not import the 'rest_api_helper' module from tests\ and crashes test_get_characters.py.

**Pull request merged**